content/docs/dvclive/api-reference/live/index.md

```diff
@@ -86,6 +86,13 @@ You can use `Live()` as a context manager. When exiting the context manager,
   [`Live.dvc_file`](/doc/dvclive/api-reference/live#properties). See
   `Live.make_dvcyaml()`. Defaults to `True`.
 
+<admon type="tip">
+
+If you have custom DVC configuration of metrics, plots, and parameters in your
+own `dvc.yaml` file, set `dvcyaml=False` to avoid duplication.
```

Comment:
WDYT about making it more explicit that the problem is only if there are overlaps? 

```suggestion
If you have are already tracking DVCLive outputs (metrics, plots, and parameters) in your
own `dvc.yaml` file, set `dvcyaml=False` to avoid duplication.
```

Reply:
```suggestion
If you are already tracking DVCLive metrics, plots, and parameters in your
own `dvc.yaml` file, set `dvcyaml=False` to avoid duplication.
```

Updated a typo and avoided the term "outputs" since they could be top-level metrics/plots/params.